README.md

```diff
@@ -165,6 +165,7 @@ models that are available out of the box.
 | guanaco-65b           |     71.8 |       1.6 |
 | vicuna-13b            |     70.4 |       1.6 |
 | oasst-rlhf-llama-33b  |     66.5 |       1.7 |
+| minotaur-13b          |     66.0 |       1.7 |
```

Comment:
we're not updating the readme leaderboard (trying to keep it only small - only `minimal` models)

Reply:
could you revert this change?

Reply:
yup! done!